sources/targets/Stride.Core.TargetFrameworks.Editor.props

```diff
@@ -2,7 +2,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <StrideEditorTargetFramework>net6.0-windows7.0</StrideEditorTargetFramework>
+    <StrideEditorTargetFramework>net8.0-windows7.0</StrideEditorTargetFramework>
```

Comment:
Cherry-pick/adapt/fix the changes from #1908, so that windows dependencies are removed for some libraries.
Or it can be done just after this PR, whichever feels safer (maybe better since it seems to have caused issues in main).

Reply:
I'm going to leave this out for this PR at least. I did try already and there was a mess of configuration I wasnt able to figure out and it broke everything when building projects.

Reply:
Fair enough. Better to not complicate this PR.